metals/src/main/scala/scala/meta/internal/metals/ScalafixProvider.scala

```diff
@@ -397,8 +425,11 @@ case class ScalafixProvider(
       val scalacOptions = {
         val list = new ju.ArrayList[String](3)
 
-        if (scalaBinaryVersion == "2.13") list.add("-Wunused:imports")
-        else list.add("-Ywarn-unused-import")
+        scalaBinaryVersion match {
+          case "3" => list.add("-Wunused:import")
+          case "2.13" => list.add("-Wunused:imports")
+          case _ => list.add("-Ywarn-unused-import")
```

Comment:
From what I see, these are just passed to make `OrganizeImports` happy, but we are not sure they are actually sent to the compiler. Unrelated to this PR, but I wonder how/if we could let the user know when this is not the case?

Reply:
> Unrelated to this PR, but I wonder how/if we could let the user know when this is not the case?

We don't really report that, might be good to do that as a follow up.